### PR TITLE
Always Throw Curveballs Option

### PR DESCRIPTION
**Pull request by Enochen**

### CODE REVIEW

**Comment by Lenovotro** [comment]
Is there a downside to always throwing curveballs? Haha

**Comment by Enochen** [comment]
It defaults to not throwing unless needed right now that's why lol

**Comment by jabbink** [comment]
>Is there a downside to always throwing curveballs?

Makes it even easier for Niantic to detect the use of bots.

File: config.properties.template [hunk 1]
```diff
@@ -69,7 +69,10 @@ autotransfer=true
 keep_pokemon_amount=1
 
 # Sort by IV first instead of CP
-sort_by_iv=false
+sort_by_iv=true
+
+# Always throw a Curveball
+always_curve=true
```
Comment (jabbink) [inline config.properties.template]: Set default to false please.

File: config.properties.template [hunk 2]
```diff
@@ -69,7 +69,10 @@ autotransfer=true
 keep_pokemon_amount=1
 
 # Sort by IV first instead of CP
-sort_by_iv=false
+sort_by_iv=true
```
Comment (jabbink) [inline config.properties.template]: Don't change the defaults of other variables

File: src/main/kotlin/ink/abb/pogo/scraper/Settings.kt [hunk 3]
```diff
@@ -77,6 +77,8 @@ class Settings(val properties: Properties) {
     val shouldAutoFillIncubatores = getPropertyIfSet("Auto Fill Incubatores", "auto_fill_incubator", true, String::toBoolean)
 
     val sortByIV = getPropertyIfSet("Sort by IV first instead of CP", "sort_by_iv", false, String::toBoolean)
+    
+    val alwaysCurve = getPropertyIfSet("Always do a Curveball", "always_curve", true, String::toBoolean)
```
Comment (jabbink) [inline src/main/kotlin/ink/abb/pogo/scraper/Settings.kt]: also here, `false`

**Comment by jabbink** [comment]
as this has merge conflicts I will put this in #399

**Pull request closed without merging**